templates/extensions/the-graph/packages/subgraph/abis/localhost_YourContract.json

```diff
@@ -0,0 +1,139 @@
+[
```

Comment:
should it be here?

Reply:
Actually, I just copy pasted `subgraph` dir from https://github.com/scaffold-eth/scaffold-eth-2/tree/subgraph-package/packages/subgraph, so it was just already present their. 

Not sure if something breaks on initial commands if we remove this, cc @kmjones1979 

or maybe we could do all this cleanup in separate PR 🙌